src/mm-repeater/mm-repeater.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+*/
+(function(scope) {
+
+	function _filterPrivateProps(inObj) {
+		var keys = Object.keys(inObj),
+			outObj = {};
+		for(var i=keys.length-1; i>=0; i--) {
+			var key = keys[i];
+			if(key.charAt(0) !== '_') {
+				outObj[key] = inObj[key];
+			}
+		}
+		return outObj;
+	}
+
+	scope.Repeater = Polymer({
+		is: 'mm-repeater',
+
+		properties: {
+			_template: {
+				type: Object,
+				value: ""
+			},
+			_model: {
+				type: Array,
+				notify: true,
+				value: function() { return []; }
+			},
+			addRowLabel: {
+				type: String,
+				value: '+Add Item'
+			}
+		},
+
+		get data() {
+			return this._model.map(_filterPrivateProps);
```

Comment:
i strongly suspect this needs to be a real property since one of the use cases for this would be *starting* with some server data (can assume its just an array of objects for simplicity) then replicating that out into the propagated form.